site/app/templates/grading/electronic/NavigationBarV2.twig

```diff
@@ -63,7 +63,7 @@
                 <button title="Toggle the two panel mode"
                         class="invisible-btn key_to_click"
```

Comment:
native HTML buttons will trigger the `onclick` even if they are clicked on with a mouse or with the keyboard, the `key_to_click` class is not required in this case